cloudant-client/src/main/java/com/cloudant/client/api/Changes.java

```diff
@@ -241,6 +241,19 @@ public Changes descending(boolean descending) {
         return this;
     }
 
+    /**
+     * Add a custom query parameter to the _changes request. Useful for specifying extra parameters
+     * to a filter function for example.
+     *
+     * @param name the name of the query parameter
+     * @param value the value of the query parameter
+     * @return this Changes instance

```

Comment:
Added tags for a bunch of things in d0b205b which I'll leave as a separate commit.